helix-core/src/comment.rs

```diff
@@ -60,7 +60,7 @@ fn find_line_comment(
 pub fn toggle_line_comments(doc: &Rope, selection: &Selection, token: Option<&str>) -> Transaction {
     let text = doc.slice(..);
 
-    let token = token.unwrap_or("//");
+    let token = token.unwrap_or("#");
```

Comment:
This feels like it should be configurable. And then change the default in the configuration.